src/Features/VisualBasic/Portable/SignatureHelp/InvocationExpressionSignatureHelpProvider.vb

```diff
@@ -116,12 +116,14 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.SignatureHelp
             Dim documentationCommentFormattingService = document.GetLanguageService(Of IDocumentationCommentFormattingService)()
 
             Dim items = New List(Of SignatureHelpItem)
-            Dim accessibleMembers = New ImmutableArray(Of ISymbol)
-            If memberGroup.Length > 0 Then
-                accessibleMembers = GetAccessibleMembers(invocationExpression, semanticModel, within, memberGroup, cancellationToken)
-                items.AddRange(GetMemberGroupItems(accessibleMembers, document, invocationExpression, semanticModel))
+            Dim accessibleMembers = ImmutableArray(Of ISymbol).Empty
+            If memberGroup.Length = 0 Then
+                Return Nothing
             End If
```

Comment:
Should this condition be moved few lines up (before or after the `memberGroup = memberGroup.Sort(semanticModel, invocationExpression.SpanStart)` line)?

Reply:
Done